### ISSUE TITLE
Fail to install v1.3 via Carthage

### ISSUE THREAD

**Issue by imroca** [JonasGessner/JGProgressHUD#37]
This is the only message i get:

Failed to read file or folder at /private/var/folders/mp/1x8f09w51rz2ghkxy3zkglwc0000gn/T/carthage-archive.w4JYoc/__MACOSX/JGProgressHUD.framework

PLS HALP!

**Comment by chrifpa** [comment]
```
±> carthage update
*** Fetching JGProgressHUD
*** Fetching ReactiveCocoa
*** Downloading JGProgressHUD.framework binary at "1.3"
Failed to read file or folder at /private/var/folders/4d/6y3_32w9591d3058fhk0gf9h0000gn/T/carthage-archive.ZNdZ6Y/__MACOSX/JGProgressHUD.framework
```

Xcode 7, iOS9 SDK, ElCap GM, Carthage 0.9.1, Deployment Target is iOS 8.2

I have only JGProgressHUD and ReactiveCocoa in my Cartfile, and ReactiveCocoa on its own works just fine. Any hint on how to solve this would be *really* appreciated.

**Comment by JonasGessner** [comment]
Fixed in 01dfbc2c3400d80e165f66d03e816b72f9ef3fff.
